cmake/NSIS.template.in

```diff
@@ -79,34 +87,34 @@
   
   Section "Freeciv21" freeciv21
     SectionIn RO 1 2
-    SetOutPath "$INSTDIR"
+    SetOutPath $INSTDIR

```

Comment:
Cannot removing quotes create issues when there are spaces in the name (does NSIS work like bash)?

Reply:
Does not impact it at all